shared/wallets/wallet/settings/display-currency-dropdown/index.desktop.tsx

```diff
@@ -43,8 +42,8 @@ const DisplayCurrencyDropdown = (props: Props) => {
         disabled={props.waiting}
         items={makeDropdownItems(props.currencies, props.selected)}
         selected={makeDropdownItem(props.selected, false, props.waiting)}
-        onChanged={(node: React.Node) => {
-          // $ForceType doesn't understand key will be string
+        onChanged={(node: React.ReactNode) => {
+          // @ts-ignore doesn't understand key will be string
           const selectedCode: Types.CurrencyCode = node.key
```

Comment:
You could also do a `as string` since you want to tell the compiler this will be a string